recipe/bld.bat

```diff
@@ -23,3 +23,8 @@ if errorlevel 1 exit /b 1
 :REM https://gitlab.com/libtiff/libtiff/-/merge_requests/338
 :REM copy "%LIBRARY_PREFIX%"\bin\tiffxx.dll "%LIBRARY_PREFIX%"\bin\libtiffxx.dll
 :REM if errorlevel 1 exit /b 1
+
+findstr /v "^[^.]*.private:" "%LIBRARY_PREFIX%"\lib\pkgconfig\libtiff-4.pc > libtiff-4.pc.new
```

Comment:
```suggestion
:REM hattne + hmaarrfk --- Aug 11, 2025
:REM https://github.com/conda-forge/libtiff-feedstock/issues/113
:REM We choose to remove the private libraries from the pkg-config file
:REM so that downstream packages can more freely depend on libtiff without
:REM also depending on its downstream dependencies
:REM We manually inspected the headers to determine that the headers of the dependencies are
:REM not exposed in the public headers of libtiff
findstr /v "^[^.]*.private:" "%LIBRARY_PREFIX%"\lib\pkgconfig\libtiff-4.pc > libtiff-4.pc.new
```